scripts/storjsimscript.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+clear
+echo "==========================="
+echo "Tool for quick storj test "
+echo "==========================="
+UPLOADFILE=""
+DOWNLOADFILE=""
+
+function installStorjRelease {
+    shutdown
+    cd $HOME
+    echo "Current Working Directory -->" $PWD 
+    if [ -d "storj" ]
+    then
+        echo "directory exists, deleting....."
+        #rm -rf storj
+    else
+        echo "directory doesn't exists"
+    fi
+
+    read -p 'Enter the Release :' storjRelease
+    echo "Downloading $storjRelease ...."
+    #git clone https://github.com/storj/storj -b $storjRelease
```

Comment:
I think we should:
1)  make a temp directory
2) make previous release directory in temp directory
3) make current release directory in temp directory
4) download/clone the release into the current release temp directory
5) download/clone the release into the previous release temp directory